### PR TITLE
LRQA-68749 | master

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LRQA-68749

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 1]
```diff
@@ -419,6 +419,42 @@ definition {
...
+
+		User.deletePersonalDataCP(
+			userFirstName = "userfn",
+			userLastName = "userln");
+
+		GDPR.deleteAndAnonymizeAllEntriesCP(itemsCount = "2");
+
+		User.openUsersAdmin();
+
+		LexiconEntry.viewNoEntry(rowEntry = "userfn userln");
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:455]: Hey @epark18! Everything looks good. Only minor change, I think these lines are already in the macro before these. Can you please remove the duplicate assertions and resend? Thanks!

Comment (epark18) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:455]: Sorry about that! Submitting changes.

**Comment by epark18** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**